gax-grpc/src/test/java/com/google/api/gax/grpc/ProtoOperationTransformersTest.java

```diff
@@ -58,35 +57,42 @@ public void testResponseTransformer() {
     OperationSnapshot operationSnapshot =
         GrpcOperationSnapshot.create(
             Operation.newBuilder().setResponse(Any.pack(inputMoney)).build());
-    Truth.assertThat(transformer.apply(operationSnapshot)).isEqualTo(inputMoney);
+    assertThat(transformer.apply(operationSnapshot)).isEqualTo(inputMoney);
   }
 
   @Test
   public void testResponseTransformer_exception() {
-    thrown.expect(UnavailableException.class);
     ResponseTransformer<Money> transformer = ResponseTransformer.create(Money.class);
     Money inputMoney = Money.newBuilder().setCurrencyCode("USD").build();
     Status status = Status.newBuilder().setCode(Code.UNAVAILABLE.value()).build();
     OperationSnapshot operationSnapshot =
         GrpcOperationSnapshot.create(
             Operation.newBuilder().setResponse(Any.pack(inputMoney)).setError(status).build());
-    Truth.assertThat(transformer.apply(operationSnapshot)).isEqualTo(inputMoney);
+    try {
+      transformer.apply(operationSnapshot);
+      Assert.fail("ResponseTransformer should have thrown an exception");
+    } catch (Exception expected) {
+      assertThat(expected).isInstanceOf(UnavailableException.class);
```

Comment:
just catch UnavailableException instead of asserting it's an instance of UnavailableException